app/controllers/api/v2/users_controller.rb

```diff
@@ -114,7 +114,11 @@ def destroy
       private
 
       def find_resource
-        editing_self? ? @user = User.find(User.current.id) : super
+        if editing_self? || params[:id] == "me"
```

Comment:
If we would do this, `User#editing_self?` would be probably better place to put the logic.